src/collectors/ntpd/ntpd.py

```diff
@@ -117,6 +117,30 @@ def get_ntpdc_stats(self):
                 data['max_error'] = val
             elif key == 'estimated error':
                 data['est_error'] = val
+            elif key == 'status':
+                data['status'] = val
+
+        return data.items()
+
+    def get_ntpdc_sysinfo_output(self):
+        return self.run_command([self.config['ntpdc_bin'], '-c', 'sysinfo'])
```

Comment:
From http://archive.ntp.org/ntp4/ChangeLog-stable if you search for `sysinfo` it seems that `sysinfo` and `kerninfo` were added at the same time. So the answer I guess is no, they should both exist.